changelog/unreleased/load-and-display-quick-actions

```diff
@@ -0,0 +1,6 @@
+Enhancement: Load and display quick actions
```

Comment:
In my understanding Enhancement is a new feature which enhances the app and Change for, well, changes.

Usually, there is Added in changelog (https://keepachangelog.com/en/1.0.0/#types) but since we haven't got this type I thought that Enhancement is its replacement.

I'm not saying I am right, just giving my thoughts about this. I just think it would be confusing to use Change for new features.

Reply:
Yeah I know, it's a pity that we don't have more than `Change` and `Enhancement`. `Added` and `Removed` would make more sense. No worries, can - of course - stay like it is.